.github/workflows/depends.yml

```diff
@@ -0,0 +1,63 @@
+name: Docker Depends Image
+
+on:
+  push:
+    branches:
+      - develop
+
+  # this runs on the develop branch
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  depends:
+    #if: github.repository == 'PecanProject/pecan'
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        R:
+          # 3.4 does not work
```

Comment:
I believe you, but am curious why not -- I was building PEcAn locally with R 3.4 until pretty recently. Too many dependencies requiring R >= 3.5?

Reply:
yeah dependencies, and instead of chasing down the dependencies, I decided to just skip it.